pkgs/development/tools/easyjson/default.nix

```diff
@@ -1,21 +1,25 @@
-{ lib, buildGoPackage, fetchFromGitHub }:
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
 
-buildGoPackage rec {
+buildGoModule rec {
   pname = "easyjson";
   version = "0.7.7";
-  goPackagePath = "github.com/mailru/easyjson";
-  goDeps = ./deps.nix;
```

Comment:
Please make sure the unnecessary `deps.nix` files are removed.

I've open a PR for `easyjson` and `goa`: https://github.com/NixOS/nixpkgs/pull/176832

Reply:
My bad. Totally missed these.